### DISCUSSION TITLE
The Overpass Turbo API

### DISCUSSION THREAD

**Discussion by mmd-osm** [steveattewell/osm-ai-map#3] [General]
Quick comment on the term "The Overpass Turbo API" you've used throughout your readme page. The proper name for the backend service is "Overpass API" (without the turbo). You've probably also used the web frontend called "overpass turbo", which sends queries to the Overpass API and visualises results on a map.

There's really no such thing as "Overpass Turbo API".

**Comment by steveattewell** [comment]
Thanks for the correction - appreciate it. I've updated the readme to remove all the "turbos" - yes I was conflating the frontend with the API. Ta
